sysctl.d/50-default.conf

```diff
@@ -36,7 +36,7 @@ net.ipv4.conf.all.promote_secondaries = 1
 #   #define GID_T_MAX (((gid_t)~0U) >> 1)
 # That's not so bad because values between 2^31 and 2^32-1 are reserved on
 # systemd-based systems anyway: https://systemd.io/UIDS-GIDS.html#summary
-net.ipv4.ping_group_range = 0 2147483647
+-net.ipv4.ping_group_range = 0 2147483647
```

Comment:
This is not compatible with sysctl anymore, and will in particular break `sysctl --system`.